Main_split_2.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include<assert.h>
+#include "Main_split_1.h"
+
+using namespace TelCoColorCoder;
+
+void testNumberToPair(int pairNumber,
+    TelCoColorCoder::MajorColor expectedMajor,
+    TelCoColorCoder::MinorColor expectedMinor)
+{
+    TelCoColorCoder::ColorPair colorPair =
+    TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+    std::cout << "Got pair " << colorPair.ToString() << std::endl;
+    assert(colorPair.getMajor() == expectedMajor);
+    assert(colorPair.getMinor() == expectedMinor);
+}
+
+void testPairToNumber(
+    TelCoColorCoder::MajorColor major,
+    TelCoColorCoder::MinorColor minor,
+    int expectedPairNumber)
+{
+    int pairNumber = TelCoColorCoder::GetPairNumberFromColor(major, minor);
+    std::cout << "Got pair number " << pairNumber << std::endl;
+    assert(pairNumber == expectedPairNumber);
+}
+
+
+int main() {
+    testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
+    testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
+
+    testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
+    testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
+
+    return 0;
```

Comment:
Somewhere, we would need the ToString method to give a string with all color pairs and their pair-numbers. I couldn't find that.

Reply:
I have done some changes Sudeep, Thanks!! for suggesting the changes I am looking forward for more improvemnt in the code

Reply:
Sudeep as suggested I have changed filenames accordingly, coverted the entire manual in a single string called final manual and computed the pair number instead of putting the limit loop number. I am enjoying doing this assignment and I am looking forward for the optimisations in the code, please suggest the same.